src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -716,9 +719,110 @@ subroutine SetServices ( GC, RC )
                                            RC=STATUS  )
   VERIFY_(STATUS)
 
+!sqz 2020----
+       ! 1: sm_surface_[forecast/analysis]           [m3 m-3]
+       ! 2: sm_rootzone_[forecast/analysis]          [m3 m-3]
+       ! 3: sm_profile_[forecast/analysis]           [m3 m-3]
+       ! 4: surface_temp_[forecast/analysis]         [K]
+       ! 5: soil_temp_layer1_[forecast/analysis]     [K]
+
+    call MAPL_AddExportSpec(GC,                    &
+    LONG_NAME          = 'surface_soil_moisture_analysis'      ,&
+    UNITS              = 'm3 m-3'                         ,&
+    SHORT_NAME         = 'SFSM_ANA'                      ,&
```

Comment:
The SHORTNAME here should be either WCSF_ANA or SFMC_ANA.
WCSF is consistent with the variable and export names in the F90 file through the LANDASSIM, LAND, and CATCH GridComps.
SFMC would be consistent with the variable name in the published lnd Collection.
Perhaps it's more consistent (and cleaner in terms of the F90) to use WCSF_ANA and then put the following line into the HISTORY.rc file:
    'WCSF_ANA'    ,  'LANDASSIM'  ,  'SFMC_ANA'
(Similarly for the other variables.)
But then again, the problem with WCSF is that the _long_ names are ridiculous from the perspective of a land modeler and urgently need cleaning up, but that's a known issue with published GEOS file specs.